src/js/videojs-dash.js

```diff
@@ -73,11 +73,18 @@ class Html5DashJS {
     // element to bind to.
     this.mediaPlayer_.initialize();
 
-    // Apply any options that are set
-    if (options.dash && options.dash.limitBitrateByPortal) {
```

Comment:
What would we warn the user of? The way to set the option isn't changing, we are just supporting more options.